src/peft/tuners/lora/model.py

```diff
@@ -367,6 +368,8 @@ def add_weighted_adapter(
         svd_clamp=None,
         svd_full_matrices=True,
         svd_driver=None,
+        density=None,
+        majority_sign_method="total",
```

Comment:
A type `Literal["total", "frequency"]` could be added to make it easier for users to see the valid values.

Reply:
Resolved in latest commit